src/renderers/testing/fiber/ReactTestFiberComponent.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactTestFiberComponent
+ * @preventMunge
+ * @flow
+ */
+
+var instanceCounter = 0;
+
+var ReactTestFiberComponent = {
```

Comment:
Note: the separation between `ReactDOMFiber` and `ReactDOMFiberComponent` is just an artifact of how it was forked from `ReactDOMComponent` and that we still want to keep them in sync when possible. So feel free to abandon that convention (e.g. see `ReactNoop.js`)